Add Azure example (#36636)
hashicorp/terraform: website/docs/language/resources/ephemeral/write-only.mdx

During a Terraform operation, the provider uses the `password_wo` value to create the database instance, and then Terraform discards that value without storing it in the plan or state file. 

> Note that the way this is written, the `password_wo` value is lost after Terraform generates unless we capture it in another resource or output. For example of generating, storing, retrieving, and using an ephemeral password as a write-only argument, refer to the [expanded example below](#examples). 

Reviewer: ```suggestion
Note that Terraform does not store the generated value for `password_wo`, but you can capture it in another resource or output. For an example of generating, storing, retrieving, and using an ephemeral password as a write-only argument, refer to [Examples](#examples). 
```
Replace "the way this is written" with more concrete language.

Reviewer 2: Does the example actually show how to do this? I thought Brian's comment in our team chat was that there's no way to capture it for Azure? What do we advise users to do instead - would they put it in their own password store manually?

Author: The example does this now, so resolving this!

Terraform first creates the secret in AWS Secrets Manager using the ephemeral `random_password`, then retrieve it using the ephemeral `aws_secretsmanager_secret_version` resource, and finally write the password to the write-only `password_wo` argument of the `aws_db_instance` resource.

### Create a MySQL database in Azure

> Another example of a using a write-only argument is to create a MySQL database in Azure. In this example, Terraform generates an password using an `ephemeral` resource and passes it to the `administrator_password_wo` write-only argument of the `azurerm_mysql_flexible_server` resource:

Reviewer: ```suggestion
You can use a write-only argument to create a MySQL database in Azure. In the following example, Terraform generates an password using an `ephemeral` resource and passes it to the `administrator_password_wo` write-only argument of the `azurerm_mysql_flexible_server` resource:
```
If someone just scanned and started reading this example, "Another example . . . " would be jarring. We should avoid assuming that someone is reading the page linearly.
